src/core/processing-queue.js

```diff
@@ -0,0 +1,155 @@
+import config from "./config";
+import {
+  defined,
+  generateHash,
+  now,
+  objectType
+} from "./utilities";
```

Comment:
Re: ESLint rule for indentation: One bit of the jQuery style guide, which the ESLint indent rule doesn't yet support, stymies this.

Specifically: Things that look like a top-level whole-file IIFE or function wrapper, but aren't pure IIFEs, are supposed to have an indent level of zero for contained statements. For example, AMD `define` callbacks, or assignments to `module.exports` that are functions, or IIFEs that follow the flexible factory pattern (the factory selector/"intro" code needs one indent level, the actual code needs zero). All of those are use cases not yet supported by ESLint.

That said, we could choose to activate the rule anyway and just accept that for this project, all function bodies will be indented by one tab. I don't think that's the end of the world to be completely honest. And both when we activate the rule and if we consume an upgraded version of the rule which supports our special function indentation preferences, we can use `eslint --fix` to fix the existing problems fairly easily.

I'll open an issue re: indent rule.